web/pdf_page_view.js

```diff
@@ -142,6 +142,25 @@ class PDFPageView {
     }
   }
 
+  /**
+   * @private
+   */
+  async _renderAnnotationLayer() {
+    let error = null;
+    try {
+      await this.annotationLayer.render(this.viewport, "display");
+      error = false;
+    } catch (e) {
+      error = e;
+    } finally {
+      this.eventBus.dispatch("annotationsrender", {
```

Comment:
This should probably be called `annotationsrendered` for consistency with existing events. (compare e.g., `textlayerrendered`).